docs/use_cases/theme.html

```diff
@@ -0,0 +1,542 @@
+                      <svg class="me-2"><use href="../images/ods-charts-icons.svg#ui-light-mode"></use></svg>
+                      Light
+                      <svg class="ms-auto d-none"><use href="../images/ods-charts-icons.svg#check2"></use></svg>
+                    </button>
+                  </li>
+                  <li>
+                    <button type="button" class="dropdown-item d-flex align-items-center" data-bs-theme-value="dark" aria-pressed="false">
+                      <svg class="me-2"><use href="../images/ods-charts-icons.svg#ui-dark-mode"></use></svg>
+                      Dark
+                      <svg class="ms-auto d-none"><use href="../images/ods-charts-icons.svg#check2"></use></svg>
+                    </button>
+                  </li>
+                  <li>
+                    <button type="button" class="dropdown-item d-flex align-items-center active" data-bs-theme-value="auto" aria-pressed="true">
+                      <svg class="me-2"><use href="../images/ods-charts-icons.svg#ui-auto-mode"></use></svg>
+                      Auto
+                      <svg class="ms-auto d-none"><use href="../images/ods-charts-icons.svg#check2"></use></svg>
+                    </button>
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <div class="title-bar">
+      <div class="container-xxl">
+        <h1 class="display-1">Theme use cases</h1>
+      </div>
+    </div>
+    <div class="container pt-3">
+      <div class="card w-100">
+        <div class="card-body">
+          <h3 class="card-title">Global <span class="text-primary">Boosted</span> theme for <u>CSS</u> and <u>popover renderer</u></h5>
+          <p class="card-text">The must common use case should be using Boosted framework and its theme. </p>
+          <p class="card-text">
+            To do so, the <code>getThemeManager()</code> has one 
+            <code>cssTheme</code> option that allow you to specify the css theme used:
+            <code>
+              <pre>
+cssTheme: ODSCharts.ODSChartsCSSThemes.BOOSTED5,
+              </pre>
+            </code>
+          </p>
+          <p>
+            When using the external popover/tooltip feature, You can also specify the javascript used to display popover or tooltip.
+            The popover/tooltip renderer can be specify by the second argument of the <code>externalizePopover()</code> feature of the theme manager:
+            <code>
+              <pre>
+        themeManager.externalizePopover({},
+          ODSCharts.ODSChartsPopoverManagers.BOOSTED5
+        );
+          </pre>
+        </code>
+          </p>
+          <p>To choose between light or dark mode, you use the <code>cssSelector</code> option of <code>getThemeManager()</code>. 
+            It must give an html selector of an HTML element where the chart is displayed:
```

Comment:
TBH, the sentence feels weird to me, maybe call it a valid selector ?
```suggestion
            It must give an HTML selector of an HTML element where the chart is displayed:
```